recipes/main/lasagne_bolognaise.adoc

```diff
@@ -15,69 +15,80 @@ image::lasagne_bolognaise/IMG_0835.jpg[Lasagne Bolognaise, 640, 480]
 Contributed by: Robert 
 image:contributors/robert_s.png[Robert, 140, 140]
 
-Lasagne Bolognaise. My way, the Bolognaise made with some additional vegetables and a lot of Béchamel. Prep time is 1.5 hours but you'll have free time while it's in the oven. 
-
-You'll need (depending on the size your baking dish):
-
-(for the Bolognaise)
-
-* 700g minced meet (pork, beef, or mixed)
-* 800g chopped Tomatoes (2 small cans)
-* tomato paste
-* 2 onions
-* small piece of celery
-* 2 carrots
-* 1 leek
-* 4 garlic cloves
-* Ketchup
-* basil
-* oregano
-* thyme
-* Tabasco
-* vegetable or meat broth
-* Salt
-* Olive oil
-
-(for the Béchamel)
-
-* 800ml Milk
-* 60g Butter
-* 3 heaped Tablespoons wheat flour
-* nutmeg
-* white pepper
-* Salt
-
-(and)
-
-* Lasagne noodles (without pre-cooking)
-* 250g grounded Cheese to gratinate
+Lasagne Bolognaise. My way, the Bolognaise made with some additional vegetables and a lot of Béchamel. Prep time is 1.5 hours but you'll have free time (to clean up the kitchen :-) while it's in the oven. 
+
+Quanitites are for 5-6 servings, you'll need (depending on the size your baking dish):
+
+.Bolognaise
+[width="100%",cols="3,3,10,10",options="header"]
+|=========================================================
+|Metric | US | Ingredient | Zutat
+
+|700g | 25 oz | Minced Meet (pork, beef, or mixed) | Hackfleisch (Schwein, Rind, oder gemischt)
+|800g | 28 oz | Chopped Tomatoes (2 small cans), or fresh Tomatoes | gehackte Tomaten (2 kleine Dosen), oder frische Tomaten
+|     |           | Tomato Paste | Tomatenmark
+|2    |           | Onions | Zwiebeln
+| 1/8 |           | Celery | Sellerie
+|2    |           | Carrots | Karotten
+|1    |           | Leek | Lauch
+|4    |           | Garlic Cloves | Knoblauchzehen
+|     |           | Ketchup | Ketchup
+|     |           | Basil | Basilikum
+|     |           | Oregano | Oregano
+|     |           | Thyme | Thymian
+|     |           | Tabasco | Tabasco
+|     |           | Vegetable or Meat Broth | Fleisch- oder Gemüsebrühe
+|     |           | Salt | Salz
+|     |           | Olive Oil | Olivenöl
+|=========================================================
+
+
+.Béchamel
+[width="100%",cols="3,3,10,10"]
+|=========================================================
+|800ml | 27 fl oz | Milk | Milch
+| 60g  | 2 oz | Butter | Butter
+| 3 heaped tbsp |  | Wheat Flour | Weizenmehl
```

Comment:
1 tbsp is fyi 15ml :wink:

Reply:
I doubt that, when it comes to "ein gehäufter Esslöffel Mehl" but found another reference that says "15g" :-)

Reply:
please check again